public/apps/configuration/constants.tsx

```diff
@@ -265,37 +279,23 @@ export const INDEX_PERMISSIONS: string[] = [
   'indices:admin/shards/search_shards',
   'indices:admin/shrink',
   'indices:admin/synced_flush',
-  'indices:admin/template/delete',
-  'indices:admin/template/get',
-  'indices:admin/template/put',
   'indices:admin/types/exists',
   'indices:admin/upgrade',
   'indices:admin/validate/query',
   'indices:data/read/explain',
   'indices:data/read/field_caps',
   'indices:data/read/field_caps*',
   'indices:data/read/get',
-  'indices:data/read/mget',
-  'indices:data/read/mget*',
-  'indices:data/read/msearch',
-  'indices:data/read/msearch/template',
```

Comment:
backend says equal to indices:data/read/msearch - should it include the /template as well